### PR TITLE
Adding some containers to the party

### PR DESCRIPTION
**Pull request by djodjo3**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,4 +49,19 @@ The first argument of the script is the path to the file you want to insert. The
...
+
+```
+docker-compose up -d
+```
+
+And you should have the application running locally.
+
+Though, we're not providing any DB at this time, so you'll have to:
+
+- Have you DB running
```
Comment (Quetzacoalt91) [inline README.md]:
```suggestion
- Have your DB running 
```

**Comment by Quetzacoalt91** [comment]
What is the issue about providing a sql server in the docker-compose?

File: README.md (hunk 1 above)
Comment (djodjo3) [reply README.md]: Good catch, I'll correct it !

**Comment by djodjo3** [comment]
No issue about providing an SQL server, it's just not done, at the moment ;)

**Pull request merged**